### PR TITLE
Demo updated

### PR DESCRIPTION
**Pull request by kayuapi**

### CODE REVIEW

File: demo/typescript/todolist.liquid [hunk 1]
```diff
@@ -1,4 +1,5 @@
-{%header content: "welcome to liquid" | capitalize%}
+{%assign title = title%}
```
Comment (harttle) [inline demo/typescript/todolist.liquid:1]: what is this line for?

**Review by harttle — CHANGES_REQUESTED** [review]

File: demo/typescript/todolist.liquid (hunk 1 above)
Comment (kayuapi) [reply demo/typescript/todolist.liquid]: oops, my bad! that line is unnecessary

**Comment by harttle** [comment]
@all-contributors please add @kayuapi  for doc

**Pull request merged**